python/ptest/cases/gomspace_checkout_case.py

```diff
@@ -15,90 +15,102 @@ def run_case_singlesat(self, simulation):
-
-        output = [
-            simulation.flight_controller.read_state("gomspace.output.output1"),
-            simulation.flight_controller.read_state("gomspace.output.output2"),
-            simulation.flight_controller.read_state("gomspace.output.output3"),
-            simulation.flight_controller.read_state("gomspace.output.output4"),
-            simulation.flight_controller.read_state("gomspace.output.output5"),
-            simulation.flight_controller.read_state("gomspace.output.output6")]
+        if vbatt < 6000 or vbatt > 8400:
+            print(
+                "Vbatt is out of expected range [6000, 8400] mV at: " + str(vbatt))
+
+        curin = [read_state(self, "gomspace.curin.output" + str(i))
+                 for i in range(1, 4)]
+
+        cursun = read_state(self, "gomspace.cursun")
+        cursys = read_state(self, "gomspace.cursys")
+
+        curout = [read_state(self, "gomspace.curout.output" + str(i))
+                  for i in range(1, 7)]
+
+        output = [read_state(self, "gomspace.output.output" + str(i))
+                  for i in range(1, 7)]
 
         for n in range(0, len(output)):
             out_n = output[n]
+            # TODO: check umbilical for which outputs should be 0 and 1
             if out_n < 0.5 or out_n > 3.0:
                 print("Output-" + n +
                       " is out of range [0.3, 0.5] at: " + out_n)
 
         wdt_i2c_time_left = simulation.flight_controller.read_state(
             "gomspace.wdt_i2c_time_left")
+        if wdt_i2c_time_left > 99:
+            print("wdt_i2c_time_left is greater than 99 seconds at: " +
+                  str(wdt_i2c_time_left))
+
         counter_wdt_i2c = simulation.flight_controller.read_state(
             "gomspace.counter_wdt_i2c")
+        print("counter_wdt_i2c is: " + str(counter_wdt_i2c))
         counter_boot = simulation.flight_controller.read_state(
             "gomspace.counter_boot")
-
-        temp = [
-            simulation.flight_controller.read_state("gomspace.temp.output1"),
-            simulation.flight_controller.read_state("gomspace.temp.output2"),
-            simulation.flight_controller.read_state("gomspace.temp.output3"),
-            simulation.flight_controller.read_state("gomspace.temp.output4")]
-
-        bootcause = simulation.flight_controller.read_state(
-            "gomspace.bootcause")
-        battmode = simulation.flight_controller.read_state("gomspace.battmode")
-        pptmode = simulation.flight_controller.read_state("gomspace.pptmode")
+        print("counter_wdt_i2c is: " + str(counter_boot))
+
+        temp = [read_state(self, "gomspace.temp.output" + str(i))
+                for i in range(1, 5)]
+        for n in range(0, len(temp)):
+            temp_n = temp[n]
+            if temp_n < 20 or out_n > 22:
+                print("Temp-" + n +
+                      " is out of room temperature range [20, 22] degC at: " + str(temp_n))
```

Comment:
Temp ends up being frequently 23-24 degrees in the lab. I think 20-25 is a safe bet.